.github/workflows/ci.yml

```diff
@@ -4,9 +4,9 @@ name: e2e tests
 
 on:
   push:
-    branches: [ master, develop ]
+    branches: [ master, develop, ipa/develop ]
   pull_request:
-    branches: [ master, develop ]
+    branches: [ master, develop, ipa/develop ]
```

Comment:
I think running for the IPA branch can be removed again

Reply:
Will be removed.